wordpress-mobile/gutenberg-mobile
# Add support for heading alignment buttons

Fixes #1880 

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/20745

This PR updates the GB reference to add support to alignment on the Heading block.
In the process, the heading block was refactored to share all the edit method with the web and that that involved using the new Text component that needed the adding of [the emotion style library](https://emotion.sh/docs/@emotion/native).
We now have support for dropdown menu in the block control. The implementation uses a BottomSheet to show the options when tapped.

<img src="https://user-images.githubusercontent.com/651601/77238492-5d3afe80-6bc8-11ea-98a4-2e419b85d6c9.gif" width=200 >


To test:
 - Open the demo app
 - Add a Heading block
 - Check if the alignment options are visible on the block toolbar
 - Check if the alignment options work correctly.
 - Check if the header size show and work correctly on the toolbar


PR submission checklist:
